draft-mdt-quic-explicit-measurements.md

```diff
@@ -464,11 +483,11 @@ This document registers a new value in the QUIC Transport Parameter Registry:
 
 Value: 0x1057 (if this document is approved)
 
-Parameter Name: loss_bits
+Parameter Name: enable_network_troubleshooting
 
-Specification: Indicates that the endpoint supports loss bits. An endpoint that
-   advertises this transport parameter can receive loss bits. An endpoint that
-   advertises this transport parameter with value 1 can also send loss bits.
+Specification: Indicates that the endpoint wants to enable the troubleshooting
+   bits. If present, both endpoints will use reserved bits as loss bits and
+   will possibly enable the latency Spin bit.
```

Comment:
Same as above -- an endpoint cannot unilaterally force another endpoint to support a protocol extension, especially since that extension may not even be implemented by the protocol.